features/192-asset-gear/index.md

```diff
@@ -0,0 +1,788 @@
+        "pattern": "^[^\\x00\\\\/:*'<>|]*$"
+      },
+      "access": { "type": "string", "enum": ["public", "private"] },
+      "headers": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "content-type": { "type": "string" },
+          "content-disposition": { "type": "string" },
+          "content-encoding": { "type": "string" },
+          "content-length": { "type": "string" },
+          "content-md5": { "type": "string" },
+          "cache-control": { "type": "string" },
+          "access-control-allow-origin": { "type": "string" },
+          "access-control-expose-headers": { "type": "string" },
+          "access-control-max-age": { "type": "string" },
+          "access-control-allow-credentials": { "type": "string" },
+          "access-control-allow-methods": { "type": "string" },
+          "access-control-allow-headers": { "type": "string" }
+        },
+        "required": ["content-length"]
+      }
+    },
+    "required": ["headers"]
+}
+```
+
+- `prefix`: The prefix to be prepended to the randomly generated asset name.
+- `access`: The access control of the asset. `public` is the default.
+- `headers`: The HTTP headers of the asset when it is retrieved.
+- `headers.content-type`: The Content-Type header. `application/octet-stream` is the default.
+- `headers.content-disposition`: The Content-Disposition header.
+- `headers.content-encoding`: The Content-Encoding header. The developer must make sure the asset is compressed in the given compression.
+- `headers.content-length`: The Content-Length header. It is required.
+- `headers.content-md5`: The [Content-MD5](https://tools.ietf.org/html/rfc1864) header. Depending on the upload method, it may be ignored.
+- `headers.cache-control`: The Cache-Control header. If it is not given, a default one is added. To remove the header, set it to an empty string.
+- `headers.access-control-*`: The CORS headers. They are only useful if the developer needs to retrieve the asset with Fetch API.
+
+##### Request Example
+
+```json
+{
+  "headers": {
+    "content-type": "image/png"
+  }
+}
+```
+
+#### Response JSON Schema
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "asset_name": { "type": "string" },
+    "url": { "type": "string" },
+    "method": { "type": "string" },
+    "headers": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": { "type": "string" },
+          "value": { "type": "string" }
+        },
+        "required": ["name", "value"]
+      }
+    }
+  },
+  "required": ["asset_name", "url", "method", "headers"]
+}
+```
+
+- `asset_name`: The asset name. It becomes valid when the asset is uploaded.
+- `url`: The presigned URL to upload the asset.
+- `method`: The method to be used in the upload request.
+- `headers`: The headers to be included in the upload request.
+
+##### Response Example
+
+```json
+{
+  "asset_name": "myimage.png",
```

Comment:
Would be great if you can put the example asset_name looks like a random string... since you specified it will be random only